src/GitVersionCore.Tests/ConfigProviderTests.cs

```diff
@@ -111,12 +112,27 @@ public void CanWriteOutEffectiveConfiguration()
     }
 
     [Test]
+    public void CanUpdateAssemblyInformationalVersioningScheme()
+    {
+        const string text = @"
+assembly-versioning-scheme: MajorMinor
+assembly-informational-format: '{NugetVersion}'";
```

Comment:
I would also like to see a test which uses multiple format values. i.e `'{Major}.{Minor}.{Patch}'`